docs/rules/no-deprecated.md

```diff
@@ -1,4 +1,8 @@
-# `import/no-deprecated`
+# import/no-deprecated
+
+💼 This rule will _warn_ in the `stage-0` config.
```

Comment:
let's exclude this undocumented config if we can :-)

Reply:
Updated using `--ignore-config stage-0`.